resources/git-branch-stage-commit.md

```diff
@@ -0,0 +1,120 @@
+# Git Basics (branch, stage, commit)
+Suppose you have just forked and cloned a repository onto your local machine. You're interested in making some changes to it, then eventually requesting that your changes be merged back into that repository. How do you get there?
+
+A good place to start is always to check the state of your files.
+
+Make sure you're in the directory that you cloned locally, then type in the command: 
+`git status`. 
+You'll see something like this:
+```
+On branch master
+Your branch is up-to-date with 'origin/master'.
+nothing to commit, working tree clean
+```
+
+Ok... cool. What does that mean? 
+
+## Branching
+### What is branching?
+When you branch, you diverge from the main line of development so that you can work on some changes without affecting the main line. 
```

Comment:
Maybe update to "When you branch, you diverge from the main line of development (usually contained on a branch named `master`)"

Reply:
Oh, I see you say that in the next paragraph... ignore me :)